doc/source/whatsnew/v1.1.0.rst

```diff
@@ -296,6 +296,7 @@ I/O
   ``coerce_timestamps``; following pyarrow's default allows writing nanosecond
   timestamps with ``version="2.0"`` (:issue:`31652`).
 - Bug in :class:`HDFStore` that caused it to set to ``int64`` the dtype of a ``datetime64`` column when reading a DataFrame in Python 3 from fixed format written in Python 2 (:issue:`31750`)
+- `read_csv` will raise a ``ValueError`` when the columns passed in `parse_dates` are missing in the dataframe (:issue:`31251`)
```

Comment:
can you use a reference for read_csv, e.g. ```:func:`read_csv` ```

Reply:
done!